custom_components/google_home/api.py

```diff
@@ -116,18 +142,20 @@ async def get_alarms_and_timers(self, device):
                         "Failed to fetch data from {device} due to invalid token. "
                         "Will refresh the token and try again."
                     ).format(
-                        device=device.device_name,
+                        device=device.name,
                     )
                 )
+                # TODO: Retry the update task
```

Comment:
Pylint doesn't like TODO comments, you can add "fixme" to the list of disabled checks.